less/shared.less

```diff
@@ -263,10 +287,7 @@ input[type="email"] {
 
 input[type="email"] {
   padding-right: 33px;
-}
-input[name="lastname"],
-input[name="zip"] {
-  padding-left: 14px;
+  margin-top: 3px;
```

Comment:
Purpose of adding 3px margin top to email?

Email input is only used on thank-you page. I don't see how rtl could cause it needing an extra 3 px top.